### ISSUE TITLE
get_user_final_grade not working

### ISSUE THREAD

**Issue by demianardus** [WIU/d2l_sdk#2]
I'm getting:

```
NameError: undefined local variable or method `le_ver' for main:Object
	from /Users/demianardus/.rvm/gems/ruby-2.3.1/gems/d2l_sdk-0.1.11/lib/d2l_sdk/grades.rb:283:in `get_user_final_grade'
```

When trying to use `get_user_final_grade`.

**Comment by MattMencel** [comment]
@demianardus we'll have a new gem version pushed out shortly

**Comment by demianardus** [comment]
Thanks!

**Comment by MattMencel** [comment]
Thanks for trying this out.  If you see any other problems or areas for improvement more PRs are welcome.  :)

**Comment by Andrew-Kulpa** [comment]
Many thanks for alerting us of this issue! All references to the versions should be fixed, now in 27475f29c2b764942d0762028439d5edbd663633.
